src/sentry/issues/utils.py

```diff
@@ -29,7 +29,7 @@ def can_create_group(
             # system-wide option
             and options.get("performance.issues.create_issues_through_platform", False)
             # more-granular per-project option
-            and project.get_option("sentry:performance_issue_create_issue_through_platform", False)
+            and project.get_option("sentry:performance_issue_create_issue_through_platform", True)
```

Comment:
@barkbarkimashark I think you might have been creating issues the old way in your draft PR because apparently, we need to pass `True` here for project options to actually enable them. Otherwise, this was always evaluated to False even if the default was set to True